Automattic/jetpack
# My Jetpack: Introduce Boost product handling class

* Introduces API endpoints to handle products
* Add tests

#### Changes proposed in this Pull Request:

* Introduces API endpoints to handle products

#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Check code
* Verify tests are passing
* Does the format of the responses look good?
